### PR TITLE
external/avs: Use wildcard to replace shell find in makefile

### PR DESCRIPTION
**Pull request by zhouxinhe**
There's such errors during make clean/distclean.
```text
find: `./avs-device-sdk/ACL/src': No such file or directory
find: `./avs-device-sdk/ADSL/src': No such file or directory
find: `./avs-device-sdk/AFML/src': No such file or directory
```
Related PR #2630 

@Taejun-Kwon @junmin-kim @jsdosa

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [a490220175413ba95482131cae097f6c0b626916] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [a490220175413ba95482131cae097f6c0b626916] - Code Rule Check (C++) OK.

**Comment by jsdosa** [comment]
@sunghan-chang

File: external/avs/Makefile [hunk 1]
```diff
@@ -206,7 +219,8 @@ depend: .depend
 clean:
 	$(call DELFILE, .built)
 	$(call CLEAN)
-	find . -name "*.o" -exec rm -rf {} \;
+	$(foreach FILE, $(CXXOBJS), $(call DELFILE, $(FILE)))
+	rm -f $(TOPDIR)/../tools/fs/contents/AlexaClientSDKConfig.json
```
Comment (jsdosa) [inline external/avs/Makefile]: @zhouxinhe 
zhou, would you add command for $ (Q)~?
The user does not need to know.

Comment (zhouxinhe) [reply external/avs/Makefile]: Done, thanks.

**Comment by seinfra** [comment]
Target : [a3eaa701d8fff15071363b1aed8121e4b6722e58] - Code Rule Check (C++) OK.

**Pull request merged**